src/Common/src/System/Net/Internals/SocketExceptionFactory.cs

```diff
@@ -18,12 +18,6 @@ public ExtendedSocketException(int errorCode, EndPoint endPoint)
                 _endPoint = endPoint;
             }
 
-            public ExtendedSocketException(SocketError socketError, int platformError)
```

Comment:
On Unix this is also used here:
https://github.com/dotnet/corefx/blob/5a37188238e90d8566e6161b43ac483a671bb7d2/src/Common/src/System/Net/Internals/SocketExceptionFactory.Unix.cs#L13